pennylane/devices/preprocess.py

```diff
@@ -287,15 +331,15 @@ def validate_adjoint_trainable_params(
     return (tape,), null_postprocessing
 
 
-@transform
+@partial(transform, plxpr_transform=preprocess_decompose_plxpr_to_plxpr)
```

Comment:
Do we really need to add the plxpr transform in this PR? Given we're not maintaining or developing the plxpr execution pipeline, this seems out of scope and unnecessary.

Reply:
https://github.com/PennyLaneAI/pennylane/pull/8225/commits/bed5f6d8820ffcfbee16d6089c97b381ee9c63ca